CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+## 2.23.0-rc.0 / 2020-11-20
+
+* [CHANGE] UI: Make the React UI default. #8142
+* [CHANGE] Remote write: The following metrics were removed/renamed in remote write. #6815
+     - `succeeded_samples_total` was removed and `samples_total` was introduced for all the samples attempted to send.
+     - `sent_bytes_total` was removed and was replaced with `samples_bytes_total` and `metadata_bytes_total`.
+     - `failed_samples_total` -> `samples_failed_total` .
+     - `retried_samples_total` -> `samples_retried_total`.
+     - `dropped_samples_total` -> `samples_dropped_total`.
+     - `pending_samples` -> `samples_pending`.
+* [CHANGE][ENHANCEMENT] Remote: Do not collect non-initialized timestamp metrics. #8060
+* [FEATURE][EXPERIMENTAL] Remote write: Allow metric metadata to be propagated via remote write. The following new metrics were introduced: `metadata_total`, `metadata_failed_total`, `metadata_retried_total`, `metadata_bytes_total`. #6815
```

Comment:
should we include the namespace of the metrics?

Reply:
or not mention the metrics

Reply:
Yes makes sense if you mean prefix them with `prometheus_remote_storage_`